{"uid":"4b4757e66a1912dae1a509f688f20b0f","children":[{"name":"Product defects","children":[{"name":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > Module build failed (from ./node_modules/ts-loader/index.js):\nError: Debug Failure. False expression: Non-string value passed to `ts.resolveTypeReferenceDirective`, likely by a wrapping package working with an outdated `resolveTypeReferenceDirectives` signature. This is probably not a problem in TS itself.\n    at Object.resolveTypeReferenceDirective (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/typescript/lib/typescript.js:43500:18)\n    at /home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/ts-loader/dist/servicesHost.js:244:56\n    at /home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/ts-loader/dist/servicesHost.js:87:142\n    at Array.map (<anonymous>)\n    at Object.resolveTypeReferenceDirectives (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/ts-loader/dist/servicesHost.js:87:125)\n    at actualResolveTypeReferenceDirectiveNamesWorker (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/typescript/lib/typescript.js:119382:163)\n    at resolveTypeReferenceDirectiveNamesWorker (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/typescript/lib/typescript.js:119682:26)\n    at processTypeReferenceDirectives (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/typescript/lib/typescript.js:121185:31)\n    at findSourceFileWorker (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/typescript/lib/typescript.js:121070:21)\n    at findSourceFile (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/typescript/lib/typescript.js:120922:26)\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","children":[{"name":"Check App build and running & Check app elements exist","uid":"517601a6b15d394a","parentUid":"683ff620e4fefa83ba39b6b84c0e074d","status":"failed","time":{"start":1681298499604,"stop":1681298499606,"duration":2},"flaky":true,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]},{"name":"Check Main Tab Functionality","uid":"3b6d26d6a5798281","parentUid":"683ff620e4fefa83ba39b6b84c0e074d","status":"failed","time":{"start":1681298499606,"stop":1681298499606,"duration":0},"flaky":true,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]},{"name":"Check Dialog elements","uid":"280e01f64bd55167","parentUid":"683ff620e4fefa83ba39b6b84c0e074d","status":"failed","time":{"start":1681298499606,"stop":1681298499606,"duration":0},"flaky":true,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]},{"name":"Check Svelte Page elements","uid":"2b23f1663b4a06c3","parentUid":"683ff620e4fefa83ba39b6b84c0e074d","status":"failed","time":{"start":1681298499606,"stop":1681298499607,"duration":1},"flaky":true,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]},{"name":"Check UI Library elements","uid":"92d3ddeaeb5d2a6c","parentUid":"683ff620e4fefa83ba39b6b84c0e074d","status":"failed","time":{"start":1681298499606,"stop":1681298499606,"duration":0},"flaky":true,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]},{"name":"Check Routing elements","uid":"97079f9c8e850830","parentUid":"683ff620e4fefa83ba39b6b84c0e074d","status":"failed","time":{"start":1681298499607,"stop":1681298499607,"duration":0},"flaky":true,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"683ff620e4fefa83ba39b6b84c0e074d"},{"name":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > Loading script failed.\n(error: http://localhost:3005/remoteEntry.js)\nwhile loading \"./components\" from webpack/container/reference/app_05\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","children":[{"name":"Check App build and running & Check app elements exist","uid":"bf031f0fa2d2097c","parentUid":"2b86580795f1bc2f0f97bac1ff09df4b","status":"failed","time":{"start":1681298603126,"stop":1681298603134,"duration":8},"flaky":true,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]},{"name":"Check Main Tab Functionality","uid":"70278f228ea0c759","parentUid":"2b86580795f1bc2f0f97bac1ff09df4b","status":"failed","time":{"start":1681298603134,"stop":1681298603134,"duration":0},"flaky":true,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]},{"name":"Check UI Library elements","uid":"2be63a2c0ec73e85","parentUid":"2b86580795f1bc2f0f97bac1ff09df4b","status":"failed","time":{"start":1681298603134,"stop":1681298603135,"duration":1},"flaky":true,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]},{"name":"Check Svelte Page elements","uid":"f7b047fa7e0f27ae","parentUid":"2b86580795f1bc2f0f97bac1ff09df4b","status":"failed","time":{"start":1681298603135,"stop":1681298603140,"duration":5},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]},{"name":"Check Dialog elements","uid":"86ae75dea5b31ee7","parentUid":"2b86580795f1bc2f0f97bac1ff09df4b","status":"failed","time":{"start":1681298603135,"stop":1681298603135,"duration":0},"flaky":true,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]},{"name":"Check Routing elements","uid":"f5a18f2d811ccbf","parentUid":"2b86580795f1bc2f0f97bac1ff09df4b","status":"failed","time":{"start":1681298603140,"stop":1681298603141,"duration":1},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"2b86580795f1bc2f0f97bac1ff09df4b"},{"name":"`cy.visit()` failed trying to load:\n\nhttp://localhost:3003/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: connect ECONNREFUSED 127.0.0.1:3003\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n\nBecause this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `Check is Comprehensive Demo...`","children":[{"name":"Check App build and running & Check app elements exist","uid":"5a22abc0e1548f15","parentUid":"27b7636b30760317b0ead2a0b8c62a59","status":"failed","time":{"start":1681298603142,"stop":1681298603142,"duration":0},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"27b7636b30760317b0ead2a0b8c62a59"},{"name":"`cy.visit()` failed trying to load:\n\nhttp://localhost:3002/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: connect ECONNREFUSED 127.0.0.1:3002\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer\n\nBecause this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `Check is Comprehensive Demo...`","children":[{"name":"Check App build and running & Check app elements exist","uid":"5980e743cc01af03","parentUid":"556fdb4de179d06a2c89480d7293d5b4","status":"failed","time":{"start":1681298603142,"stop":1681298603142,"duration":0},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[]}],"uid":"556fdb4de179d06a2c89480d7293d5b4"}],"uid":"8fb3a91ba5aaf9de24cc8a92edc82b5d"},{"name":"Test defects","children":[{"name":"The following error originated from your application code, not from Cypress.\n\n  > Module build failed (from ./node_modules/ts-loader/index.js):\nError: Debug Failure. False expression: Non-string value passed to `ts.resolveTypeReferenceDirective`, likely by a wrapping package working with an outdated `resolveTypeReferenceDirectives` signature. This is probably not a problem in TS itself.\n    at Object.resolveTypeReferenceDirective (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/typescript/lib/typescript.js:43500:18)\n    at /home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/ts-loader/dist/servicesHost.js:244:56\n    at /home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/ts-loader/dist/servicesHost.js:87:142\n    at Array.map (<anonymous>)\n    at Object.resolveTypeReferenceDirectives (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/ts-loader/dist/servicesHost.js:87:125)\n    at actualResolveTypeReferenceDirectiveNamesWorker (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/typescript/lib/typescript.js:119382:163)\n    at resolveTypeReferenceDirectiveNamesWorker (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/typescript/lib/typescript.js:119682:26)\n    at processTypeReferenceDirectives (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/typescript/lib/typescript.js:121185:31)\n    at findSourceFileWorker (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/typescript/lib/typescript.js:121070:21)\n    at findSourceFile (/home/runner/work/module-federation-examples/module-federation-examples/comprehensive-demo-react18/app-05/node_modules/typescript/lib/typescript.js:120922:26)\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","children":[{"name":"Check App build and running & Check app elements exist","uid":"d64bed86e0f94bfe","parentUid":"0f50fdd4cb379083c974d4c8bf33ed41","status":"broken","time":{"start":1681298500172,"stop":1681298500655,"duration":483},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":true,"retriesCount":1,"retriesStatusChange":true,"parameters":[]}],"uid":"0f50fdd4cb379083c974d4c8bf33ed41"},{"name":"`cy.visit()` failed trying to load:\n\nhttp://localhost:3005/\n\nWe attempted to make an http request to this URL but the request failed without a response.\n\nWe received this error at the network level:\n\n  > Error: connect ECONNREFUSED 127.0.0.1:3005\n\nCommon situations why this would fail:\n  - you don't have internet access\n  - you forgot to run / boot your web server\n  - your web server isn't accessible\n  - you have weird network configuration settings on your computer","children":[{"name":"Check App build and running & Check app elements exist","uid":"ed968a35f00180bf","parentUid":"a7bbac9a7b251ce021f5bca55ccd3942","status":"broken","time":{"start":1681298604968,"stop":1681298606251,"duration":1283},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":true,"retriesCount":1,"retriesStatusChange":true,"parameters":[]}],"uid":"a7bbac9a7b251ce021f5bca55ccd3942"}],"uid":"bdbf199525818fae7a8651db9eafe741"}],"name":"categories"}